kpi/urls/router_api_v2.py

```diff
@@ -150,6 +152,13 @@ def get_urls(self, *args, **kwargs):
                        ServiceUsageViewSet, basename='service-usage')
 router_api_v2.register(r'users', UserViewSet)
 
+
+# Merge django apps routers with API v2 router
+# All routes are under `/api/v2/` within the same namespace.
+router_api_v2.registry.extend(language_router.registry)
```

Comment:
If we are adding this, can we then remove `path('', include(router.urls)),` in audit_log.urls and languages.urls? It seems like the URL is being added twice.

And also remove 

```
    re_path(r'^api/v2/', include('kobo.apps.languages.urls')),
    re_path(r'^api/v2/', include('kobo.apps.audit_log.urls')),
```